src/components/widgets/EditableText.js

```diff
@@ -48,6 +49,12 @@ class EditableText extends Component {
     }
   }
 
+  handleWheel(event) {
+    if (this.props.onWheel && document.activeElement === this._ref) {
+      this.props.onWheel(event);
+    }
```

Comment:
oh.. so you could affect the behavior even when component wasn't rendered yet..?

Reply:
this is to prevent the onWheel to take over if the input isn't focused... the problem was that it would 'grab' the event if the mouse was just hovered over the input field, so it made just normal scrolling be annoying!